src/Framework/Http/Response/Contracts/ResponseFactoryInterface.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Give\Framework\Http\Response\Contracts;
+
+use Give\Framework\Http\Response\Response;
+use Give\Framework\Http\Response\Types\JsonResponse;
+use Give\Framework\Http\Response\Types\RedirectResponse;
+
+/**
+ * @unreleased
+ */
+interface ResponseFactoryInterface
```

Comment:
I was going off what Laravel did - I think they made it a contract so it can be used in a facade or something. I'm not sure if that's completely necessary for us.

Reply:
Laravel is built such that most everything can be overridden via the Service Container. I don't want to encourage that with GiveWP as we're not a framework. 😄

Reply:
not yet that is... lol I removed it 766bd7a8